tests/ui/fmt/format-string-wrong-order.stderr

```diff
@@ -0,0 +1,18 @@
+error: invalid format string: expected format parameter to occur after `:`
+  --> $DIR/format-string-wrong-order.rs:3:15
+   |
+LL |     format!("{?:}", bar);
+   |               ^ expected `?` to occur after `:` in format string
+   |
+   = note: if you intended to format, try `:?` instead
```

Comment:
This is great!

I'd change the wording slightly for the note: "`?` comes after the `:`", and if we could it'd be amazing if we could use a `span_suggestion_verbose` for this, so that people can see what the code should have been.

Reply:
In regards to the note, I can change the wording, no problem. We should still keep the suggestion to fix it, right? 

I have no idea what a `span_suggestion_verbose` is. I cannot find it in the `rustc_parse_format`. Does it still apply to here?